test/local_integration_test.py

```diff
@@ -214,3 +230,12 @@ def check_bundles_are_indexed(self, test_name: str, entity_type: str, indexed_bu
                             f'There was a problem during indexing an {entity_type} entity'
                             f' {hit["entryId"]}. Bundle(s) ({",".join(bundle_fqids)})'
                             f' have been indexed without the debug project name. Contains {project}')
+
+    def _get_file_uuid(self, lambda_endpoint: str, url: str):
+        """Returns file UUID of smallest fastq file in query."""
+        url = lambda_endpoint + url
+        response = self.requests.get(url)
+        response.raise_for_status()
```

Comment:
And please stop duplicating functionality. There already is a function that does exactly those three lines.

Reply:
Rewrote it. Now uses returned response from def `check_endpoint_is_working`.